.circleci/config.yml

```diff
@@ -0,0 +1,18 @@
+version: 2
+jobs:
+  build:
+    working_directory: ~/circleci-java
+    docker:
+      - image: circleci/openjdk:8-jdk-stretch
+
+    steps: 
+      - checkout
+      - restore_cache: # restore the saved cache after the first run or if `pom.xml` has changed
+          key: circleci-demo-java-spring-{{ checksum "pom.xml" }}
```

Comment:
Why `demo`?

Reply:
It was copy&pasted from the example, the key doesn't matter.

Reply:
I can imagine it does not matter, but it would be nice to make it name it somehow meanigful IMO (: fine for now.